Solutions/PrismaCloudCompute/Data Connectors/connectorDefinition.json

```diff
@@ -0,0 +1,117 @@
+{
+    "name": "PrismaCloudComputeDefinition",
+    "apiVersion": "2022-09-01-preview",
```

Comment:
id property is mandate. check for api supportability of the key in clv2

Reply:
done